controllers/secretgenerator/secretgenerator_controller.go

```diff
@@ -132,54 +132,54 @@ func (r *SecretGeneratorReconciler) Reconcile(ctx context.Context, request ctrl.
 	generatedSecretKey := types.NamespacedName{
 		Name: generatedSecret.Name, Namespace: generatedSecret.Namespace,
 	}
-	err = r.Client.Get(ctx, generatedSecretKey, generatedSecret)
-	if err != nil {
-		if k8serr.IsNotFound(err) {
-			// Generate secret random value
-			log.Info("Generating a random value for a secret in a namespace",
-				"secret", generatedSecret.Name, "namespace", generatedSecret.Namespace)
 
-			secret, err := NewSecretFrom(foundSecret.GetAnnotations())
-			if err != nil {
-				log.Error(err, "error creating secret %s in %s", generatedSecret.Name, generatedSecret.Namespace)
-				return ctrl.Result{}, err
-			}
+	getErr := r.Client.Get(ctx, generatedSecretKey, generatedSecret)
+	if getErr != nil && k8serr.IsNotFound(getErr) {
+		// Generate secret random value
+		log.Info("Generating a random value for a secret in a namespace",
+			"secret", generatedSecret.Name, "namespace", generatedSecret.Namespace)
 
-			generatedSecret.StringData = map[string]string{
-				secret.Name: secret.Value,
-			}
+		secret, err := NewSecretFrom(foundSecret.GetAnnotations())
```

Comment:
this `err` variable is a new variable, which was shadowing the one from `r.Client.Get` before it got renamed.